src/vs/workbench/browser/parts/editor/editorParts.ts

```diff
@@ -700,6 +700,68 @@ export class EditorParts extends MultiWindowParts<EditorPart> implements IEditor
 		if (groupScopedContextKeys) {
 			this.scopedContextKeys.delete(group.id);
 		}
+
+		const registeredContextKeys = this.registeredContextKeys.get(group.id);
```

Comment:
This entire methods can be simplified to just call `delete` without checking if the group is there.